### DISCUSSION TITLE
RESTEasy  create  temp files

### DISCUSSION THREAD

**Discussion by huangfc** [quarkusio/quarkus#40394] [Community]
how to avoid create temp files when upload file by RESTEasy

**Comment by codespearhead** [comment]
What Quarkus version are you using? Because #34039 seems to fix the issue of temporary files managing the content of the parts not being deleted automatically.

The alternative's holding everything in memory, but that's awful in terms of performance and stability when handling large files or multiple files simultaneously, which exacerbates the impact of DDoS attacks.

Possible duplicates: #33986 .

**Reply by huangfc** [reply]
resteasy create temp file is delete when i use deleteUploadFilesonEnd is true but it is bad for Disk IO，I use quarkus version 3.5.2;

**Comment by geoand** [comment]
Please provide more details about the extensions you are using, the details under which you are seeing the temp file being created and the Quarkus configuration you are using.
